server/embed/config.go

```diff
@@ -329,6 +329,9 @@ type Config struct {
 	// ExperimentalWarningUnaryRequestDuration is the time duration after which a warning is generated if applying
 	// unary request takes more time than this value.
 	ExperimentalWarningUnaryRequestDuration time.Duration `json:"experimental-warning-unary-request-duration"`
+	// ExperimentalWatchableStoreErrorReporting enables WatchableStore to report error in case of failure. Normally we
+	// treat these failures as panics.
+	ExperimentalWatchableStoreErrorReporting bool `json:"experimental-watchable-store-error-reporting"`
```

Comment:
This seems too specific to internal packages. How about `report-kv-decode-error`? /cc @ptabor